docs/agreement-creation/agreement-processing.md

```diff
@@ -18,18 +16,73 @@ An agreement's PDF is processed with the following goals:
 - have the ability to prove that a page is contained within a PDF without sharing the original PDF
 - have the ability to prove that a form field is contained within a PDF without sharing the original PDF
 
-In order to accomplish this we create two sparse merkle trees (SMTs): an SMT that represents the pages of a PDF
+To accomplish this two sparse merkle trees (SMTs) are created: an SMT that represents the pages of a PDF
```

Comment:
Worth linking `sparse merkle tree` to something like https://wiki.polygon.technology/docs/zkEVM/zkProver/sparse-merkle-tree/ ?